src/test/java/com/findmyflight/findmyflight/UnitTest.java

```diff
@@ -0,0 +1,8 @@
+package com.findmyflight.findmyflight;
+
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+public interface UnitTest {
```

Comment:
We should create that kind of "config class" for integration tests also - name it `IntegrationTest`, it should contain all necessary annotations for integration tests. Then, when writing tests for `EmailNotificationReceiverController`, create a test class and extend it with `IntegrationTest` class - this way we'll have the integration tests config in one place and could easily add more classes with integration tests in the future.